### PR TITLE
Blog app - controllers specs

### PR DESCRIPTION
**Pull request by Salwa99**
# Blog app - controllers specs

### Hello there! :wave: 

**In this PR I:**

- Create unit tests for `Users` actions.

- Create unit tests for `Posts` actions.

- Fix linter errors.

- Follow GitFlow.

Cheers! :heart:

### CODE REVIEW

File: spec/requests/posts_spec.rb
Comment (fobadara) [inline spec/requests/posts_spec.rb:1]: [REQUIRED] Kindly ensure that the Spec file includes tests that ensure the response body includes the correct placeholder text.
Example : expect(response.body).to include("Welcome to user index page")

File: spec/requests/users_spec.rb
Comment (fobadara) [inline spec/requests/users_spec.rb:1]: [REQUIRED] Kindly ensure that the Spec file includes tests that ensure the response body includes the correct placeholder text.
Example : expect(response.body).to include("Welcome to user index page")

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @Salwa99, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing
✅ Good checking for the right HTTP status codes in specs

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: spec/requests/posts_spec.rb
Comment (Salwa99) [reply spec/requests/posts_spec.rb:1]: Thank you @fobadara, I'm gonna make sure to add them

**Review by OLIPLICHE — APPROVED** [review]
Hi @Salwa99, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights
- Wrote tests for controllers👏
- Applied the changes requested by the previous review 👍 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Salwa99** [comment]
Thank you @OLIPLICHE :smile_cat:

**Pull request merged**